master/pkg/actor/actor.go

```diff
@@ -30,3 +30,10 @@ type Actor interface {
 	// a request to stop is received or the parent shuts the actor down.
 	Receive(context *Context) error
 }
+
+// ActorFunc is a function that encapsulates behavior. It is a stateless actor, useful for a mocking.
```

Comment:
```suggestion
// ActorFunc is a function that encapsulates behavior. It is a stateless actor, useful for mocking.
```

Reply:
🤦 somehow I didn't see this. I can fix this typo in my next PR.